src/tests/integration/D3MCompound.t.sol

```diff
@@ -0,0 +1,1225 @@
+// SPDX-FileCopyrightText: © 2021-2022 Dai Foundation <www.daifoundation.org>
```

Comment:
Same as above for tests, consider keeping file creation year (for compound it should be 2022) and removing old copyright comment (as per Dai Foundation copyright guidance).

Reply:
Sounds good.